<launch>
	<!-- <rosparam command="load" file="$(find viobot)/config/viobot.yaml" />   -->
    <node name="optical_flow_test" pkg="optical_flow_test" type="optical_flow_test" output="screen" respawn="false" launch-prefix="xterm -e gdb -ex run --args">
        <!-- <param name="tks_pro_integration" type="bool" value="true" /> -->
        <!-- <param name="src_ip_address" type="string" value="192.168.55.28" /> -->
        <!-- <param name="dest_ip_adderss" type="string" value="192.168.55.21" /> -->
        <!-- <param name="debug_mode" type="bool" value="false" /> -->
        <!-- <param name="output_data_file" type="string" value="/home/ita560/docker/data_output/" /> -->
        <!-- <param name="data_output" type="bool" value="true" /> -->
        <!-- <param name="data_display" type="bool" value="true" /> -->
        <!-- <param name="bag_flag" type="bool" value="true" /> -->
        <!-- <param name="atp_id" type="int" value="0" />  -->
        <!-- <param name="odom_calc_sec_flag" type="bool" value="true" /> -->
        <param name="sys_yaml_file" type="string" value="$(find optical_flow_test)/config/sys_baton.yaml" />
        <!-- <param name="sys_yaml_file" type="string" value="$(find stereo3)/config/sys.yaml" /> -->
        <param name="diff_pose_path" type="string" value="$(find optical_flow_test)/../../Log/diff_pose.csv" />
        <param name="config_path" type="string" value="$(find optical_flow_test)/" />
    </node>
    <!-- <node name="visualizer" pkg="rviz" type="rviz" output="screen" args="-d $(find optical_flow_test)/config/circle.rviz" /> -->
</launch>
